azurerm/internal/services/bot/parse/bot_channel_email.go

```diff
@@ -0,0 +1,33 @@
+package parse
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type BotChannelEmailId struct {
+	ResourceGroup string
+	BotName       string
+}
+
+func BotChannelEmailID(input string) (*BotChannelEmailId, error) {
+	id, err := azure.ParseAzureResourceID(input)
+	if err != nil {
+		return nil, fmt.Errorf("[ERROR] Unable to parse Bot Channel Email ID %q: %+v", input, err)
+	}
+
+	service := BotChannelEmailId{
+		ResourceGroup: id.ResourceGroup,
+	}
+
+	if service.BotName, err = id.PopSegment("botServices"); err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	}
	
	if service.Name, err = id.PopSegment("channels"); err != nil {
		return nil, err
	}
```

Reply:
(it might not be worth exposing this field, but we should error if it's not what we expect too)